rcon/broadcast.py

```diff
@@ -36,22 +34,28 @@ def __str__(self):
             return self.default
 
 
-def get_votes_status(none_on_fail=False):
+def get_votes_status(none_on_fail=False) -> VoteOverview | None:
     return VoteMap().get_vote_overview()
 
 
-def format_winning_map(ctl, winning_maps, display_count=2, default=None):
-    nextmap = ctl.get_next_map()
+def format_winning_map(
+    ctl: Rcon,
+    winning_maps: Sequence[tuple[maps.Layer, int]],
+    display_count=2,
+    default=None,
+):
+    nextmap: str = ctl.get_next_map()
     if not winning_maps:
         if default:
             return str(default)
         return f"{nextmap}"
     wins = winning_maps[:display_count]
     if display_count == 0:
         wins = winning_maps
-    return ", ".join(
-        f"{LONG_HUMAN_MAP_NAMES[m]} ({count} vote(s))" for m, count in wins
-    )
+
+    # Example warfare map: Carentan Warfare (2 vote(s))
+    # Example offensive map: Driel Off. AXIS (2 vote(s))
+    return ", ".join(f"{map_} ({num_votes} vote(s))" for map_, num_votes in wins)
```

Comment:
`Layer.__str__` returns the layer's ID, not a prettified version. Use `map_.pretty()` instead.

Reply:
Also no idea how this doesn't fail the test you wrote for this? Perhaps I'm wrong somehow?

Reply:
No it fails, I think I wrote that part before I made some other changes? I don't remember but I definitely did not re-run all the tests before opening a PR (whoops lol).

Fixed.